coverage/PENDING_REVIEW/intel/0423-COVERAGE-nir-opt-if-833/cov-if-true-continue.amber

```diff
@@ -0,0 +1,235 @@
+#!amber
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# A test for a coverage-gap found by the GraphicsFuzz project.
+
+# Short description: A fragment shader that covers specific NIR code paths
+
+# The test passes because the shader always writes red.
+
+SHADER vertex variant_vertex_shader PASSTHROUGH
+
+# variant_fragment_shader is derived from the following GLSL:
+# #version 320 es
+# #define _int_0 _GLF_uniform_int_values[0]
+# #define _int_10 _GLF_uniform_int_values[1]
+# #define _int_1 _GLF_uniform_int_values[2]
+# #define _int_2 _GLF_uniform_int_values[3]
+#
+# precision highp float;
+# precision highp int;
+#
+# // Contents of _GLF_uniform_int_values: [0, 10, 1, 2]
+# layout(set = 0, binding = 0) uniform buf0
+# {
+#     int _GLF_uniform_int_values[4];
+# };
+# layout(location = 0) out vec4 _GLF_color;
+#
+# void main()
+# {
+#     int a = _int_0;
+#     int b = _int_10;
+#     int c = _int_1;
+#
+#     while (a < b)
+#     {
+#         a++;
+#
+#         if (c == _int_1)
+#         {
+#             c *= _int_2;
+#         }
+#         else if (true)
+#         {
+#             continue;
+#         }
+#     }
+#
+#     if (a == b && c == _int_2)
+#     {
```

Comment:
I think we could add some comments here like:

Loop iterates 10 times. `a` goes from 0 to 10. `b` is 10. `c` goes from 1 to 2 on first iteration.

True only on first iteration.

Always true.

Reply:
@Vihanakangas When a comment has been addressed please mark as resolved. Then when you're ready for a re-review, reply on the main conversation with a comment @-ing the person who did the first review and requested changes.